src/OpenApi/src/Services/OpenApiDocumentService.cs

```diff
@@ -0,0 +1,25 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.Extensions.Hosting;
+using Microsoft.OpenApi.Models;
+
+namespace Microsoft.AspNetCore.OpenApi;
+
+internal class OpenApiDocumentService(IHostEnvironment hostEnvironment)
+{
+    private readonly string _defaultOpenApiVersion = "1.0.0";
```

Comment:
Static?

Reply:
Actually, given that it's private, why wouldn't it be const?

Reply:
Yep, it can be.